.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
```

Comment:
It's a huge image (2.58 GB): isn't there a smaller alternative?

Reply:
No, the Workspaces don't really exist smaller, unfortunately.